templates/index.html

```diff
@@ -2,33 +2,45 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+
+    <!-- This will change the address bar's theme for chrome android. -->
+    <meta name="theme-color" content=":#212121" />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
     <title>Put Bernie Anywhere!</title>
+    <link rel="stylesheet" href="C:\Users\DELL\Desktop\Programs\New folder\bernie-sits\static\styles.css">
```

Comment:
You should fix this link to use relative path to the css file

Reply:
I forgot to update it. Thank you